### PR TITLE
PR | Issue-36 | Main layout for workflow page implemented

### PR DESCRIPTION
**Pull request by VladislavGulik**

### CODE REVIEW

File: build/src/workflow.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="main__row horizontal row">
+                <div class="main__col col">
+                    <span></span>
+                    <span></span>
+                    <a href=""></a>
+                    <a href="">
+                        <img src="" alt="">
+                    </a>
+                    <div class="main__col col_tab">
+                        <img src="" alt="photo">
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: You have to put css classes for every element on this page

File: build/src/workflow.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                        <span></span>
+                        <a href="">
+                            <img src="" alt="">
+                        </a>
+                    </div>
+                </div>
+                <div class="main__col col_dragndrop-tab">
+                    <span></span>
+                    <span></span>
+                    <a href=""></a>
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: this one should be a .link (builitin component)

File: build/src/workflow.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    <a href="">
+                        <img src="" alt="">
+                    </a>
+                    <div class="main__col col_tab">
+                        <img src="" alt="photo">
+                        <span></span>
+                        <span></span>
+                        <a href="">
+                            <img src="" alt="">
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: And this one .image (builtin component)

File: build/src/workflow.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                <div class="main__col col_dragndrop-tab">
+                    <span></span>
+                    <span></span>
+                    <a href=""></a>
+                    <a href="">
+                        <img src="" alt="">
+                    </a>
+                    <div class="main__col col_tab">
+                        <img src="" alt="photo">
+                        <span></span>
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: regulaar text on page that not created by RichTextArea (wysiwyg editor) should be marked as .txt (builtin component)

File: build/src/workflow.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Workflow</title>
+</head>
+<body>
+    <main class="main main_workflow-page">
+        <div class="main__container">
+            <div class="main__row horizontal row">
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: Are you using some library for project?
It seems to me we don't need `row` here. And `horizontal` too.
But you can save your decision (waiting for arguments);

File: build/src/workflow.html [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+                    <div class="main__col col_dragndrop-tab">
+                        <img src="" alt="photo">
+                        <span></span>
+                        <span></span>
+                        <a href="">
+                            <img src="" alt="">
+                        </a>
+                    </div>
+                </div>
+                <div class="main__col col_dragndrop-tab">
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: What is `col_dragndrop-tab`, i don't understand. We have to discuss it.

File: build/src/workflow.html [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+                    <span></span>
+                    <span></span>
+                    <a href=""></a>
+                    <a href="">
+                        <img src="" alt="">
+                    </a>
+                    <div class="main__col col_dragndrop-tab">
```
Comment (arsenykonohovwork) [inline build/src/workflow.html]: Following our discussion `.col_mod` (block modifier) should be placed with `.col` (block).

**Comment by arsenykonohovwork** [comment]
@VladislavGulik please follow my notions above;

**Comment by VladislavGulik** [comment]
I fix the code for this PR. Check out last commit and approve if everything is fine, please.

**Pull request merged**